### DISCUSSION TITLE
Consume message from activeMQ by postman / message listener with postman

### DISCUSSION THREAD

**Discussion by Hugoman67** [postmanlabs/postman-flows#108] [Q&A] [answered]
Hello,
I have created a small flow in postman and I would like to start the same as soon as a message (ActiveMQ) I have been subscribed to can be consumed. Is there a possibility to realize this with postman? I. e. by establishing a listener?

Thanks a lot in advance and best regards, Michael

**Comment by samay-v** [comment, marked as answer]
@Hugoman67 Currently, you can only trigger a flow manually from the canvas. But having multiple asynchronous triggers to a flow is on our roadmap and you should be able to achieve this usecase soon
